recipe/build_package.sh

```diff
@@ -1,24 +1,56 @@
 #!/usr/bin/env bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* ./Cbc
-cp $BUILD_PREFIX/share/gnuconfig/config.* .
 set -e
 
-UNAME="$(uname)"
-export CFLAGS="${CFLAGS} -O3"
-export CXXFLAGS="${CXXFLAGS} -O3"
-export CXXFLAGS="${CXXFLAGS//-std=c++17/-std=c++11}"
+# LIBRARY_PREFIX will only be available on Windows
+if [ ! -z ${LIBRARY_PREFIX+x} ]; then
+    USE_PREFIX=$LIBRARY_PREFIX
+else
+    USE_PREFIX=$PREFIX
+fi
 
-# Use only 1 thread with OpenBLAS to avoid timeouts on CIs.
-# This should have no other affect on the build. A user
-# should still be able to set this (or not) to a different
-# value at run-time to get the expected amount of parallelism.
-export OPENBLAS_NUM_THREADS=1
+if [[ "${target_platform}" == win-* ]]; then
+  #COINUTILS_LIB=( --with-coinutils-lib='${LIBRARY_PREFIX}/lib/mkl_intel_ilp64.lib ${LIBRARY_PREFIX}/lib/mkl_sequential.lib ${LIBRARY_PREFIX}/lib/mkl_core.lib ${LIBRARY_PREFIX}/lib/libCoinUtils.lib' )
+  #COINUTILS_INC=( --with-coinutils-incdir='${LIBRARY_PREFIX_COIN}' )
+  #OSI_LIB=( --with-osi-lib='${LIBRARY_PREFIX}/lib/libOsi.lib' )
+  #OSI_INC=( --with-osi-incdir='${LIBRARY_PREFIX_COIN}' )
+  CLP_LIB=( --with-clp-lib='${LIBRARY_PREFIX}/lib/libClp.lib' )
+  CGL_INC=( --with-clp-incdir='${LIBRARY_PREFIX_COIN}' )
```

Comment:
Still some inconsistency here cgl vs clp

Reply:
Ah, OK, I missed this, although this was not what was causing any of the issues, since all the headers are in the same directory anyway.